templates/cloudflare-pages/server.ts

```diff
@@ -0,0 +1,13 @@
+import { logDevReady } from "@remix-run/cloudflare";
+import { createPagesFunctionHandler } from "@remix-run/cloudflare-pages";
+import * as build from "@remix-run/dev/server-build";
+
+if (process.env.NODE_ENV === "development") {
+  logDevReady(build);
```

Comment:
This logs too soon for me resulting in a failed request from the browser after HMR updates because the server is not yet listening on the port when it's restarted by wrangler. By adding a `setTimeout` things work fine but definitely is hacky (I'm on 100ms to be safe).

Reply:
Per [my comment on #6415](https://github.com/remix-run/remix/issues/6415#issuecomment-1553969445), I think this is only a problem in Wrangler v2. The appropriate fix for this should probably be to upgrade the dependency to v3.

Reply:
You're right, it works great without `setTimeout` with Wrangler v3, so definitely recommend updating the dev dependency for the templates. Thanks!